sdks/rust/src/protocol/mod.rs

```diff
@@ -47,7 +47,7 @@ pub fn compute_namespace_registry_address(namespace: &str) -> Result<Vec<u8>, Ad
         }
     };
     let hash = sha512_hash(prefix.as_bytes());
-    Ok([&parse_hex(NAMESPACE_REGISTRY_ADDRESS_PREFIX)?, &hash[..64]].concat())
+    Ok([&parse_hex(NAMESPACE_REGISTRY_ADDRESS_PREFIX)?, &hash[..32]].concat())
```

Comment:
Could these prefix's be just byte array constants?

Reply:
They can, but would require a new minor version release.

Reply:
Added the prefixes as byte arrays and used them where possible.